lua/noirbuddy/init.lua

```diff
@@ -1,17 +1,28 @@
 local M = {}
 
+M.options = nil
+
 function M.setup(opts)
-  opts = opts or {}
+  M.options = opts
+end
 
+function M.load(opts)
+  opts = vim.tbl_extend("force", M.options, opts or {})
+
+  vim.api.nvim_command("highlight clear")
```

Comment:
Added "highlight clear" – I found that when switching to noirbuddy after first activating another colorscheme, this was needed to remove left over highlights from the other colorscheme.